WindowsServerDocs/administration/server-core/server-core-removed-roles.md

```diff
@@ -23,20 +23,24 @@ The following roles, role services, and features have been removed from the Serv
 - Fax Server (**Fax**)
 - MultiPoint Services (**MultiPointServerRole**)
 - Network Policy and Access Services (**NPAS**)
-- Windows Deployment Services (**WDS**) *(before Windows Server version 1803)*
+- Windows Deployment Services (**WDS**) *
 
 ## Role Services not in Server Core
-Note that some Remote Desktop role services are included in Server Core (Connection Broker, Licensing, Virtualization Host), but others are NOT (Gateway, RD Session Host, Web Access).
+> [!NOTE]
+Remote Desktop Licenses services are still included in Server Core. Other features have been removed after Windows Server 1803 SAC / Windows Server 2019 LTSC.
+This removal does not affect the availability in Windows Server SAC or LTSC prior to the named releases.
```

Comment:
Just to make sure that this Note blob formatting does not get misread by MarkDown or the HTML conversion, we use the same indent marker for any and all of the subsequent lines belonging to this Note:
```suggestion
> Remote Desktop Licenses services are still included in Server Core. Other features have been removed after Windows Server 1803 SAC / Windows Server 2019 LTSC.
> This removal does not affect the availability in Windows Server SAC or LTSC prior to the named releases.
```